{
  "totalNumberSkippingTests": 62,
  "totalMemory": 363855872,
  "bugType": "only_fail",
  "workspace": "./workspace",
  "totalNumberErroringTests": 0,
  "repo": "druid-io/druid",
  "freeMemory": 154808776,
  "bugCommit": "https://github.com/druid-io/druid/compare/b9fd30e90ad9...5a5a2749cdc5",
  "failingModule": "/root/workspace/druid-io/druid/226817632/server",
  "buildid": 226817632,
  "hostname": "repairnator",
  "totalNumberFailingTests": 4,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 27422,
  "failing-test-cases": [
    {
      "className": "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest",
      "failingMethods": [
        "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest#testRetryAllFail",
        "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest#testRetrySuccessAtSecondLocation"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "isError": false
        }
      ],
      "nbFailures": 2,
      "nbErrors": 0
    },
    {
      "className": "io.druid.segment.loading.LocalDataSegmentPusherTest",
      "failingMethods": [
        "io.druid.segment.loading.LocalDataSegmentPusherTest#testPushCannotCreateDirectory"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Expected test to throw (an instance of java.io.IOException and exception with message a string containing \"Unable to create directory\")",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "io.druid.query.lookup.LookupSnapshotTakerTest",
      "failingMethods": [
        "io.druid.query.lookup.LookupSnapshotTakerTest#testIOExceptionDuringLookupPersist"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Expected exception: io.druid.java.util.common.ISE",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ],
  "nbCPU": 4,
  "bugRepo": "druid-io/druid"
}